src/main/pageobjects/shopByCategory.page.ts

```diff
@@ -0,0 +1,44 @@
+import Page from "./page";
+import * as config from "config";
+
+//const baseURL = config.get("Environment.baseUrl");
+
+export default class ShopByCategory extends Page {
+  public get shopBy() {
+    return <any>$("//h2[text()='Shop by Category']");
+  }
```

Comment:
Please separate two methods or properties by adding a new line.

Reply:
public get brandText() {
    return <any>$("//span[.='Brands']");
  }
  public get boatBrand() {
    return <any>$("//span[.='boAt']");
  }


should be :

public get brandText() {
    return <any>$("//span[.='Brands']");
  }

  public get boatBrand() {
    return <any>$("//span[.='boAt']");
  }